main/inc/lib/message.lib.php

```diff
@@ -2260,6 +2260,92 @@ public static function getMessagesFromLastReceivedMessage($userId, $lastId = 0)
         return $messages;
     }
 
+    /**
+     * Get the data of the last received messages for a user.
+     *
+     * @param int $userId The user id
+     * @param int $lastId The id of the last received message
+     *
+     * @return array
+     */
+    public static function getReceivedMessages($userId, $lastId = 0)
```

Comment:
`MessageManager::getMessagesFromLastReceivedMessage` returns the last unread messages.

What's the difference with `MessageManager::getMessagesFromLastReceivedMessage`?

Reply:
Hi Angel,
MessageManager::getReceivedMessages this function return all received message (read & unread), I also add picture of sender.

Reply:
:+1: